modules/heat_conduction/src/userobjects/GapFluxModelPressureDependentConduction.C

```diff
@@ -0,0 +1,71 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "GapFluxModelPressureDependentConduction.h"
+
+registerMooseObject("HeatConductionApp", GapFluxModelPressureDependentConduction);
+
+InputParameters
+GapFluxModelPressureDependentConduction::validParams()
+{
+  InputParameters params = GapFluxModelBase::validParams();
+
+  params.addClassDescription("Heat flux model across a closed gap to calculate the conductance "
+                             "between two solid materials");
+  params.addCoupledVar("temperature", "The name of the temperature variable");
```

Comment:
Required? It appears it's pulled below in the constructor without asking for its existence.